google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/StatementParser.java

```diff
@@ -460,6 +464,12 @@ static String removeStatementHint(String sql) {
       startQueryIndex = upperCaseSql.indexOf(keyword);
       if (startQueryIndex > -1) break;
```

Comment:
nit: I know this was not changed in this PR, but please apply the following as well, as the Google style guide dictates that all `if` statements should always use curly braces.

```suggestion
      if (startQueryIndex > -1) {
        break;
      }
```

Reply:
Done